Move-a-WebGL-1-Engine-To-WebGL-2-Blog-1.md

```diff
@@ -233,8 +233,8 @@ gl.drawArrays(gl.TRIANGLES, 0, 6);
 
 ## Fragment Depth
 
-The fragment shader can explicitly control the depth value for the current fragment.
-This operation can be expensive because it forces the GPU to bypass its normal early-out fragment discard behavior. 
+The fragment shader can explicitly set the depth value for the current fragment.
+This operation can be expensive because it can force disable early-z optimizations. 
```

Comment:
I would change to "it can cause the early-z optimization to be disabled." You can link perhaps this https://software.intel.com/en-us/articles/early-z-rejection-sample or some other article.